### PR TITLE
Remove wots from bee-message

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change
 
Remove WOTS

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes
- [ ] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

File: bee-message/src/payload/transaction/essence.rs [hunk 1]
```diff
@@ -201,25 +201,22 @@ impl TransactionEssenceBuilder {
         let mut total: u64 = 0;
 
         for output in self.outputs.iter() {
-            match output {
```
Comment (thibault-martinez) [inline bee-message/src/payload/transaction/essence.rs:204]: Any specific reason for this change ? I'll need to handle dust very soon so I'll have to put it back

File: progress.txt [hunk 2]
```diff
@@ -24,8 +24,7 @@
 │   │       ├── output
```
Comment (thibault-martinez) [inline progress.txt]: What the hell is this file

File: bee-message/src/payload/transaction/essence.rs (hunk 1 above)
Comment (Thoralf-M) [reply bee-message/src/payload/transaction/essence.rs:204]: Clippy warning

File: progress.txt (hunk 2 above)
Comment (Thoralf-M) [reply progress.txt]: You tell me

Comment (thibault-martinez) [reply progress.txt]: Did I do that ahah ?

Comment (thibault-martinez) [reply progress.txt]: I did 🤦

File: bee-message/src/payload/transaction/essence.rs (hunk 1 above)
Comment (thibault-martinez) [reply bee-message/src/payload/transaction/essence.rs:204]: Alright, I guess I'll bring it back, all good

File: progress.txt (hunk 2 above)
Comment (thibault-martinez) [reply progress.txt]: Can you remove it ?

Comment (Thoralf-M) [reply progress.txt]: Yes

**Pull request merged**